impl/client/src/main/javascript/web/pentaho/visual/config.js

```diff
@@ -826,6 +826,22 @@ define(function() {
         }
       },
 
+      // Waterfall
+      {
+        priority: RULE_PRIO_VIZ_DEFAULT,
+        select: {
+          module: "pentaho/ccc/visual/Waterfall"
+        },
+        apply: {
+          extension: {
+            rootCategoryLabel: "",
+            legendAreaVisible: true,
+            colorMode: "level",
+            axisGrid: true,
+          }
```

Comment:
Consider specifying default CCC options in the View class' `_options` property, e.g. https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/e0f7e113f12c3fd0d8631794c1f5d131fdf5e6b0/impl/client/src/main/javascript/web/pentaho/ccc/visual/BarLine.js#L40.

Reply:
For waterfall, `axisGrid` seems to be over-written by https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/impl/client/src/main/javascript/web/pentaho/visual/config.js#L372

Reply:
Oh, good point. Overriding those effectively requires a configuration rule, so I'm ok if you leave these here.